BrainPortal/config/console_rc/lib/interactive_bourreau_control.rb

```diff
@@ -157,7 +157,10 @@ def process_user_letter(letter) #:nodoc:
         W = starts workers
         T = stops workers
         U = stops workers and waits to make sure
-        K = stops bourreaux
+        K = stops bourreaux (using shell commands)
+        J = stops workers and bourreaux (using command control
+            messages; use this only if you're sure no workers
+            are active and the SSH masters are opened)
```

Comment:
Oh, I see, so still dangerous, but only done by an experienced admin.

Reply:
Well, an admin who can run the 'ps' command first.

Reply:
and fast one :)

(seems fine thought theoretically with same effort some signaling or mutex mechanism can be implemented to avoid the issue)